cmd/operator/api/v1alpha1/kubearchiveconfig_webhook.go

```diff
@@ -84,3 +92,33 @@ func (kac *KubeArchiveConfig) validateCELExpressions() (admission.Warnings, erro
 	}
 	return nil, errors.Join(errList...)
 }
+
+//nolint:unparam
+func (kac *KubeArchiveConfig) validateSingleton() (admission.Warnings, error) {
+	cfg := config.GetConfigOrDie()
+	// Use a rest.WarningHandler that discards warning messages.
+	cfg.WarningHandler = rest.NoWarnings{}
+
+	scheme := runtime.NewScheme()
+	err := AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	c, err := client.New(cfg, client.Options{Scheme: scheme})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I think that the idea of using the client provided by the sigs library is a good choice but I feel that something here is not right, that we shouldn't be mixing the `k8s.io` library with the `sigs.k8s.io` package... I'm trying to look for some examples of the usage of this client in other projects

Reply:
@maruiz93 , does my last change make you feel better?  Please check it out.

Also, this may become irrelevant as I plan to do https://github.com/kubearchive/kubearchive/issues/585 next.